server/views/taskList/tasks/caSubmitAddressReviewTask.pug

```diff
@@ -4,12 +4,14 @@ div.largeMarginBottom
 
     div.pure-g
         div.pure-u-1.pure-u-md-3-4
-            if ['STARTED', 'DONE'].includes(licenceStatus.tasks.curfewAddress)
+            if licenceStatus.decisions.optedOut
+                | Submission unavailable - Offender has opted out of HDC
+            else if ['STARTED', 'DONE'].includes(licenceStatus.tasks.curfewAddress)
                 | Ready to submit
             else
                 | Submission unavailable - Tasks not yet complete
 
         div.pure-u-1.pure-u-md-1-4.center
-            if ['STARTED', 'DONE'].includes(licenceStatus.tasks.curfewAddress)
+            if ['STARTED', 'DONE'].includes(licenceStatus.tasks.curfewAddress) && !licenceStatus.decisions.optedOut
```

Comment:
This could be lifted up and done together before the opted-in bits

if opted out
    div.pure-g
        blah offender has opted out
else
    div.pure-g
        all original blah for opted in case